js/reducers/approval.js

```diff
@@ -62,20 +61,20 @@ module.exports = function approval(state, action) {
 		case actions.FAIL_APPROVAL_SUBMIT:
 			return state;
 
-		case actions.START_APPROVAL_CANCEL:
+		case actions.SUCCEED_APPROVAL_CANCEL:
 			return _.assign({}, state, {
 				request_sent: false,
 				requested_time: '',
 				approved: false,
```

Comment:
I wonder if we should roll `approve`, `bypass` and `rejected` into a singular field together with the deployment.. basically just "expose" the deploy state in the frontend?